addons/mail/static/src/web/activity/activity_cell.xml

```diff
@@ -5,7 +5,7 @@
         <div class="h-100 cursor-pointer p-3" t-on-click="onClick">
             <div class="h-100 d-flex align-items-center justify-content-center position-relative" t-ref="content">
```

Comment:
Hello @khpa-odoo 

Thank you for your PR :smiley: If we want to avoid the overlap, wouldn't it be better to get rid of the "position: absolute" and let the elements take their natural size? The elements will then never overlap with each other, no matter their sizes.

We could for instance replace the `position-relative` class here with `gap-1` and remove the custom css rules you updated. You can maybe add a `text-nowrap` class on the cell as well. What do you think?